src/block-components/helpers/borders/deprecated.js

```diff
@@ -0,0 +1,43 @@
+import { addFilter } from '@wordpress/hooks'
+import { dispatch } from '@wordpress/data'
+
+addFilter( 'stackable.block-component.helpers', 'borders', ( output, getAttribute, updateAttributes ) => {
+	const borderRadius = getAttribute( 'borderRadius' )
+	const borderRadiusTablet = getAttribute( 'borderRadiusTablet' )
+	const borderRadiusMobile = getAttribute( 'borderRadiusMobile' )
+	dispatch( 'core/block-editor' ).__unstableMarkNextChangeAsNotPersistent()
```

Comment:
`__unstableMarkNextChangeAsNotPersistent` applies to the **next** `updateAttributes` call. Right now this is called with the possibility that 0-3 `updateAttributes` would be called.

I recommend combining all of the attributes to be changed into just one object (all the changes in just one call) also.